### PR TITLE
Screen/rules

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Added rules screen

### CODE REVIEW

**Review by manojncsoni007 — COMMENTED** [review]
LGTM

**Comment by vermasavita** [comment]
Please add cdn or netflify link

File: category.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    </header>
+    <main class="container h-full w-full col items-center">
+        <section class="category-section w-full text-light">
+            <h1 class="text-3xl text-bold mb-4 text-center">Choose <span class="text-secondary">Trivia</span> Category
+            </h1>
+            <div class="row flex-gap-1 justify-center">
+                <div class="card w-60 bg-primary">
+                    <div class="card-header">
+                        <img src="images/classical.png" alt="" class="img h-40 w-full cover">
+                    </div>
```
Comment (Ankur9669) [inline category.html:26]: fix: alt tag should not be empty

File: category.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                    <div class="card-bottom">
+                        <div class="card-body">
+                            <div class="card-title">Classical Music</div>
+                        </div>
```
Comment (Ankur9669) [inline category.html:30]: fix: You can use p tag for card title because it would be semantic

File: category.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                        <div class="card-footer">
+                            <a href="rules.html" class="w-full"><button class="btn btn-dark w-full">Play</button></a>
+                        </div>
+                    </div>
+                </div>
+                <div class="card w-60 bg-primary">
+                    <div class="card-header">
+                        <img src="images/edm.png" alt="" class="img h-40 w-full cover">
+                    </div>
```
Comment (Ankur9669) [inline category.html:39]: Same alt tag is empty

File: category.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+                    <div class="card-bottom">
+                        <div class="card-body">
+                            <div class="card-title">Electronic Dance Music</div>
+                        </div>
```
Comment (Ankur9669) [inline category.html:43]: fix: You can use p tag for card title because it would be semantic

File: category.html [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+                    </div>
+                </div>
+                <div class="card w-60 bg-primary">
+                    <div class="card-header">
+                        <img src="images/hiphop.png" alt="" class="img h-40 w-full cover">
+                    </div>
+                    <div class="card-bottom">
+                        <div class="card-body">
+                            <div class="card-title">Hiphop Music</div>
+                        </div>
```
Comment (Ankur9669) [inline category.html:56]: fix: You can use p tag for card title because it would be semantic

**Review by Ankur9669 — COMMENTED** [review]
Most of the code looks great, i have suggested some changes doing that would make your code even better

**Pull request merged**